src/tracing/SwapEventTimestampTracker.ts

```diff
@@ -0,0 +1,51 @@
+import { calculateElapsedTimeWithPerformanceMark } from './utils'
+
+export enum SwapEventType {
+  FIRST_SWAP_ACTION = 'FIRST_SWAP_ACTION',
+  FIRST_QUOTE_FETCH_STARTED = 'FIRST_QUOTE_FETCH_STARTED',
+  FIRST_SWAP_SIGNATURE_REQUESTED = 'FIRST_SWAP_SIGNATURE_REQUESTED',
+  FIRST_SWAP_SIGNATURE_COMPLETED = 'FIRST_SWAP_SIGNATURE_COMPLETED',
```

Comment:
qq: these 3 events aren't used other than for tests, are they to be implemented later?

Reply:
yes, i added them all at once to the names package but upcoming PRs will start using them in interface